src/main/java/frc/robot/Constants.java

```diff
@@ -34,4 +34,35 @@ public static enum Mode {
     /** Replaying from a log file. */
     REPLAY
   }
+
+  public static final class canIDs {
```

Comment:
CAN IDs are specific to device type so this should be something like `SparkMaxCANIDs`